datalad/support/external_versions.py

```diff
@@ -251,5 +255,14 @@ def dumps(self, indent=None, preamble="Versions:", query=False):
             out += " " + ' '.join(items)
         return out
 
+    def require(self, name, min_version=None, msg=""):
```

Comment:
This new method looks around and sees that all its neighbors have beautiful docstrings.  It stares out at the night sky and wonders, "Why do I exist? What's my purpose in all of this?"

Reply:
oh, I made it into a bastard... sorry -- will fix now